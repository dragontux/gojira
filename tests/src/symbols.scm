; symbols
(display 'a)
(newline)

(display 'abc)
(newline)

(display (eq? 'abc 'abc))
(newline)

(display (eq? 'abc 'def))
(newline)

(display 'zeblarg)
(newline)

(display 'fooba&r-_baz?)
(newline)

(display :asdf)
(newline)

(display (eq? :asdf ':asdf))
(newline)
