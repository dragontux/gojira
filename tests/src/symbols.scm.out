a
abc
#t
#f
zeblarg
fooba&r-_baz?
:asdf
#t
